events/e_2_clients_lite_nodes.toml

```diff
@@ -1,5 +1,5 @@
 # the name of your track or event
-name = "Browsers and Mobile"
+name = "Clients and Light Nodes"
```

Comment:
Let's mix this track entirely, if we need it we can create in a follow up

Reply:
can you clarify - what do you mean?

Reply:
ah, did you mean nix? if so, why? it's the highest priority for the stuff my team is working on atm.

Reply:
Apologies, I did originally mean "nix" (still getting used to reviewing PRs from a phone 😵‍💫).

I'm committed to talks & attendance on the data transfer / data agony tracks on Thursday, making it hard for me to lead this track.

Unrelated, but I'm wary of adding another track with no planned speakers, but my hunch is this is just a matter of coordination, and once you show me the light @autonome it'll all make sense ☺️

Any chance we could move this track to a separate PR & discuss there? I think all the changes to the Friday track are great, and could be merged

Reply:
gotcha, will do